docs/CppInteroperability/CppStdlibOverlay.md

```diff
@@ -0,0 +1,126 @@
+# C++ Standard Library Overlay
+
+To refine and improve ergonomics of the C++ standard library classes when they
+are used from Swift code, Swift provides an overlay for the C++ standard
+library. The overlay is written in Swift, and is shipped as part of the OS SDKs
```

Comment:
This is true for macOS only, for Linux/Windows it's just shipped alongside the Swift standard library. Maybe you can reword it?

Reply:
You're right, I'll rephrase this.
Although I'd say this is correct on Windows: Swift on Windows has a similar installation directory structure to Darwin, where SDKs are located outside of the toolchain. However, we don't ship the overlay on Windows at the moment anyway.